src/R/Components/Impl/PackageManager/Implementation/RPackageManager.cs

```diff
@@ -59,24 +59,6 @@ internal class RPackageManager : IRPackageManager {
             return await GetPackagesAsync(async (eval) => await eval.AvailablePackages());
```

Comment:
That's something to consider if people feel it's not fast enough.  If I have package manager opened in my last VS session, when I start VS it opens in the "installed" tab, I immediately click on "available" and it's about 3 secs before all available show up.